MyCapytain/resources/prototypes/metadata.py

```diff
@@ -9,18 +9,25 @@
 
 from MyCapytain.common.metadata import Metadata
 from MyCapytain.errors import UnknownCollection
-from MyCapytain.common.utils import Subgraph, literal_to_dict
```

Comment:
Why did you break this import up? They are both coming from the same location.

Reply:
Mistake